openfisca_core/simulations.py

```diff
@@ -238,10 +236,7 @@ def _run_formula(self, variable, population, period):
         if formula is None:
             return None
 
-        if self.trace:
-            parameters_at = self.trace_parameters_at_instant
-        else:
-            parameters_at = self.tax_benefit_system.get_parameters_at_instant
+        parameters_at = self.trace_parameters_at_instant
```

Comment:
Reset to previous code as it's not easy to move `trace_parameters_at_instant` into `tracers.py` (needs the tax_benefit_system).
It could be improved in an other PR.